vendor/modules.txt

```diff
@@ -69,7 +69,7 @@ github.com/sirupsen/logrus/hooks/test
 ## explicit; go 1.13
 github.com/stretchr/testify/assert
 github.com/stretchr/testify/require
-# github.com/stripe/goproxy v0.0.0-20231206175114-560c3ba6a2a1
+# github.com/stripe/goproxy v0.0.0-20240702232545-72d7dbc6d4fe
```

Comment:
Did we bring in something new from `goproxy`?

Reply:
yep! a bunch of these changes are there; I'll send you the PR